api/instance.go

```diff
@@ -36,6 +35,7 @@ func (api *API) waitUntilAllNodesReady(instanceID string) error {
 	failed := make(map[string]interface{})
 
 	for {
+		time.Sleep(30 * time.Second)
```

Comment:
30 seconds make sense when creating an instance, but when resizing disk, we could poll much quicker... not sure what a good trade of would be, maybe lower this to 15 seconds? Should be fine when creating instances

Reply:
Looks like creating instance uses waitUntilReady with a sleep of 10 seconds. While update instance use waitUntilAllNodesReady with 30 seconds. 

However tried out to make the sleep configurable. Make it possible to keep 30 seconds for update instance and 15 seconds for resizing disk.
`func (api *API) waitUntilAllNodesReady(instanceID string, sleep time.Duration) error {`